presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcDeletedRows.java

```diff
@@ -229,24 +247,26 @@ private static Path createPath(AcidInfo acidInfo, AcidInfo.DeleteDeltaInfo delet
             return new Path(directory, fileName.substring(0, fileName.lastIndexOf("_")));
         }
 
+        if (acidInfo != null && acidInfo.getOriginalFiles().size() > 0) {
+            /**
+             * construct delete delta file path from bucket ID.
+             */
```

Comment:
Javadoc cannot be placed inline, so make this an ordinary comment:

```suggestion
            // construct delete delta file path from bucket ID.
```

Also, the comment describes **what** we are doing (and this is apparent from the code anyway).
Please enhance it with **why** we are doing this.

Reply:
fixed.